rpcs3/evdev_joystick_handler.h

```diff
@@ -1,4 +1,4 @@
-#pragma once
```

Comment:
If I remember right I had to set VS2017 to save as UTF8 without signature, otherwise it would add signature for my local codepage. It's done automatically.

Reply:
But UTF8 without signature should leave the files without the BOM mark...

Reply:
Just checked and the file is now encoded with UTF-8-BOM which should be fine?